### ISSUE TITLE
Creating and Training a custom dataset.

### ISSUE THREAD

**Issue by HyperLethalVector** [alexgkendall/caffe-posenet#18]
Hi,

I'm a little new to Caffe. (So sorry for the newbie questions)

How can I train a new dataset using your posenet architecture?
I wish to benchmark PoseNet against several other localization systems, but this is a little confusing for me.
(I can easily assume that using weights from another dataset will lead to terrible results).

Can you provide a quick guide of how you created your weights and models? 

Thanks for your time.

**Comment by DRAhmadFaraz** [comment]
@HyperLethalVector Are you able to train the pose-net on your own custom images.?

**Comment by HyperLethalVector** [comment]
Yes after a fair amount of trial and error, and reading a lot of caffe tutorials online. 

------
From the software-ronin
Damien Rompapas.

PhD Student.
Nara Institute of Science and Technology
Interactive Media Design Laboratory.
 
Facebook: https://www.facebook.com/ryu.uesugi
Mail: damien.rompapas.dk1@is.naist.jp
Google Mail: hyperlethalvector92@gmail.com
Web:  www.denkikitchen.com

**Comment by DRAhmadFaraz** [comment]
@HyperLethalVector Respected Sir, Thanx for replying me, 
Can you please guide me how to run this posenet code on our own Custom RGB Images from basic steps.?
I will be so thankful to you.
Best Regards

**Comment by DRAhmadFaraz** [comment]
@HyperLethalVector How to get the Landmarks in this format of text file for our own dataset.? as shown

[https://github.com/alexgkendall/caffe-posenet/blob/master/posenet/7Scenes_PoseNet_TrainVal/chess_train.txt](url)

**Comment by DRAhmadFaraz** [comment]
@HyperLethalVector can you please tell me, how to calculate the poses using visual SFM for the posenet Grount Truth..??
